From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Choong, Yin Thong" <yin.thong.choong@intel.com>
Date: Tue, 2 Jul 2019 14:50:28 +0000
Subject: [PATCH] Enable build link with system sqlite3 lib

Signed-off-by: Choong, Yin Thong <yin.thong.choong@intel.com>
---
 Makefile.PL | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index 8771fe1..0bbac12 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -128,8 +128,8 @@ SCOPE: {
 # the system SQLite. We expect that anyone sophisticated enough to use
 # a system sqlite is also sophisticated enough to have a patching system
 # that can change the if ( 0 ) to if ( 1 )
-my ($sqlite_local, $sqlite_base, $sqlite_lib, $sqlite_inc);
-if ( 0 ) {
+my ($sqlite_local, $sqlite_base, $sqlite_lib, $sqlite_inc) = ("","","/usr/lib64","/usr/include");
+if ( 1 ) {
 	require File::Spec;
 	if ( $sqlite_base = (grep(/SQLITE_LOCATION=.*/, @ARGV))[0] ) {
 		$sqlite_base =~ /=(.*)/;
